### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by H-CODE6** [himashi92/VT-UNet#19]
Hello, author, through your paper, I found that you divided 1251 pieces of data into training, verification and testing for experiments, rather than taking part in the 21 challenge to obtain 251 verification sets for experiments. Is this a convincing division?

**Comment by himashi92** [comment]
Hello, we evaluated all the methods using the same data split for a fair comparison. Yes, in our future works, we hope to evaluate results using Brats continuous evaluation platform.

**Comment by H-CODE6** [comment]
Hello, I randomly divided the data set for testing, and found that the TC index is greater than WT during the evaluation. Is this reasonable.

**Comment by H-CODE6** [comment]
We look forward to your reply

**Comment by himashi92** [comment]
Hello, What are the values you got?

**Comment by H-CODE6** [comment]
Hello, I am a test set randomly divided by myself. During the evaluation, the TC index is about 0.001-0.005 higher than the WT index. The same is true for tests on other models. Is this because of my randomly divided test set? I look forward to your reply.

**Comment by H-CODE6** [comment]
For example, wt=0.9086, tc=0.9098

**Comment by himashi92** [comment]
yes, that might be the case.

**Comment by H-CODE6** [comment]
OK, thank you very much. I used the same test set to test on multiple models. The TC index is greater than the WT index. Then I can judge whether each model is good or bad by the size of this index

**Comment by H-CODE6** [comment]
We look forward to your reply

**Comment by H-CODE6** [comment]
OK, thank you very much. I used the same test set to test on multiple models. The TC index is greater than the WT index. Then I can judge whether each model is good or bad by the size of these coefficients?

**Comment by himashi92** [comment]
yes, You can evaluate all three tumor classes (i.e., ET, TC, and WT) and get the average dice coefficient of the three classes and rank the models.

**Comment by H-CODE6** [comment]
Hello, author, are the other comparative experimental results shown in your paper all obtained under the same epoch?

**Comment by H-CODE6** [comment]
We look forward to your reply

**Comment by himashi92** [comment]
Hello, not under the same epoch. Results were shown for the best model saved after evaluating model performance on the validation dataset.

**Comment by H-CODE6** [comment]
Hello, I found that many models run hundreds of epochs and save the last model. At this time, how can we judge that this is the best model? In this case, it is difficult to judge how many epochs are the best performance in the comparative experiment.

**Comment by H-CODE6** [comment]
In this case, it is difficult for us to obtain the optimal performance of each model in how many epochs. Can we compare the performance under the same epoch?

**Comment by himashi92** [comment]
During training, we use a validation dataset to identify the best model that would be performed well for unseen test cases. It is the recommended method in the machine learning process. Once you save the best model (You can evaluate your model after every epoch during training and save the model), then you can evaluate your model on the test dataset. That's why we split our dataset into, training, validation, and test. If you set an epoch, say max_epoch=100, how can you say that your model is converged or not? Some models can be overfitted after the 70th epoch, so getting the model at the 100th epoch for all the baseline models will not be a fair comparison.

**Comment by himashi92** [comment]
if you are following someone's paper, then its better to do experiments in their way.

**Comment by H-CODE6** [comment]
Well, thank you very much.
